### ISSUE TITLE
Change on epic?

### ISSUE THREAD

**Issue by silentguy256** [vogler/free-games-claimer#300]
Has there been a change to the epic page? Starting last thursday I keep getting 

`epic-games failed: page.click: Timeout 60000ms exceeded.`

**Comment by 4n4n4s** [comment]
Same for me @vogler

**Comment by Helios747** [comment]
yeah I think Epic is starting to put in some anti automation measures. I don't get any capcha on official client or browser, I do under free-games-claimer.

**Comment by MrrZed0** [comment]
i have it running in windows 11 and it was working now it just says 'got a captcha during login likely due to too many attempts try a new ip or try in a few hours. this is on a new install and the old install where i was already logged in says the same when trying to grab any games
